src/compiler/crystal/tools/formatter.cr

```diff
@@ -3650,8 +3650,8 @@ module Crystal
         skip_space
         check :OP_EQ
         next_token_skip_space_or_newline
```

Comment:
We can go a tiny step further by not dropping the newline token:

```suggestion
        next_token_skip_space
```

Then the formatter will accept:

```crystal
property foo =
  if something
  else
  end
```

Reply:
Done!